nix/pkgs/status-go/library/default.nix

```diff
@@ -0,0 +1,62 @@
+{ 
+  pkgs ? import <nixpkgs> { },
```

Comment:
`<nixpkgs>` is not reproducable and probably wont work with flakes or will not work in the future.